vector/src/main/res/values/strings.xml

```diff
@@ -1165,6 +1165,9 @@
 
     <!-- String for Play Store -->
     <string name="store_title">Riot.im - open team collaboration</string>
+    <string name="store_whats_new">"We’re always making changes and improvements to Riot.im.
+The complete changelog can be found here: %1$s.
```

Comment:
it's just a plain text? or %1$s is a link to an external resource?

Reply:
I will replace by the link when updating play store resource, there is no need to put the link here